src/@codemirror__view/internals/Vim/VimApi.d.ts

```diff
@@ -1,5 +1,7 @@
-import type { VimEditor } from './VimEditor.d.ts';
-import type { VimState } from './VimState.d.ts';
+import type { ActionArgsPartial } from 'types';
+// import type { VimEditor } from './VimEditor.d.ts';
+// import type { VimState } from './VimState.d.ts';
+import type { VimStateVim as vimState } from './VimStateVim.js';
```

Comment:
use aliases only when they are conflicting with the typings you are trying to write